source/_components/image_processing.facebox.markdown

```diff
@@ -58,3 +65,29 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## {% linkable_title Automations %}
+
+Use the `image_processing.detect_face` events to trigger automations, and breakout the `trigger.event.data` using a [data_template](https://www.home-assistant.io/docs/automation/templating/). The following example automation sends a notification when Ringo Star is recognised:
+
+{% raw %}
+```yaml
+- id: '1120092824666'
+  alias: Ringo Starr recognised
+  trigger:
+  - event_data:
+      name: 'Ringo_Starr'
+    event_type: image_processing.detect_face
+    platform: event
+  condition: []
+  action:
+  - data_template:
+      message: Ringo_Starr recognised with probability {{ trigger.event.data.confidence }}
+      title: Door-cam notification
+    service: notify.platform
```

Comment:
Same, adjust the sequence. `service` comes before `data_template`

Reply:
ok